assets/javascripts/discourse/lib/block-kit.js

```diff
@@ -0,0 +1,70 @@
+function stringToDOMNode(string) {
```

Comment:
I think we should add this to the core discourse/lib/utilities, we have added the exact same function in a core test and we've also made similar functionality in other places in core (though not inside a dedicated function)

https://github.com/discourse/discourse/blob/1c9a0fe18e19831277fd1c1be6d28559b3bad108/app/assets/javascripts/discourse/tests/unit/lib/formatter-test.js#L13-L18

https://github.com/discourse/discourse/blob/1c9a0fe18e19831277fd1c1be6d28559b3bad108/app/assets/javascripts/pretty-text/addon/oneboxer.js#L78-L80

https://github.com/discourse/discourse/blob/1c9a0fe18e19831277fd1c1be6d28559b3bad108/app/assets/javascripts/discourse/tests/unit/lib/link-mentions-test.js#L44-L47

Reply:
Yes I know I already have a PR awaiting merge on this: https://github.com/discourse/discourse/pull/15310 ;)

Reply:
Oh nice! I saw you couldn't add to discourse-common either, a pain I know too well :joy: